### DISCUSSION TITLE
새로운 스탯 코드?

### DISCUSSION THREAD

**Discussion by ghost** [storycraft/node-kakao#175] [General]
-328이라는 스탯코드가 발견되었는데 노드카카오에는 없는거같아 이슈 남겨봅니다.
로그인할때 (아마도 LOGINLIST) 스탯코드 -328이 나오면 disconnect 후 이용자 보호조치 걸리는것으로 지금까지 확인되었습니다.

**Comment by storycraft** [comment]
해당 스탯코드는 차단된 서브디바이스에서 로그인시 반환 되는것으로 확인됩니다

**Comment by storycraft** [comment]
Issues와 관계없는 글 Discussion으로 이동
